Friendly-plans/app/src/main/java/pg/autyzm/friendly_plans/utils/Validation.java

```diff
@@ -0,0 +1,79 @@
+package pg.autyzm.friendly_plans.utils;
+
+import android.widget.EditText;
+import java.util.regex.Pattern;
+
+/***
+ * Original of validation class is here:
+ * https://tausiq.wordpress.com/2013/01/19/android-input-field-validation/
+ */
+public class Validation {
+
+    // Regular Expression
+    private static final String NAME_REGEX = "^[a-zA-Z0-9_. -]*$";
+    private static final String NUM_ONLY_REGEX = "^[0-9]*$";
+
+    // Error Messages
+    private static final String WRONG_NAME_MSG = "Only letters and numbers are allowed";
```

Comment:
I would prefer to store error messages in string.xml file that we could easily translate all app in the future.